### PR TITLE
chore: modify failing e2e test

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes

[Failing e2e test related to S3 change](https://app.circleci.com/pipelines/github/aws-amplify/amplify-cli/19362/workflows/4a6d45f1-a6b8-489d-afae-ddfa91fb4dfb/jobs/928962), this will permit us to get meaningful feedback from the test while accounting for S3 bucket permission changes.
#### Description of how you validated changes
yarn test, e2e test run

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by sundersc — APPROVED** [review]

**Review by alharris-at — DISMISSED** [review]

**Review by goldbez — DISMISSED** [review]

File: packages/amplify-e2e-tests/src/schema-api-directives/tests/predictions-usage.ts [hunk 1]
```diff
@@ -55,16 +55,21 @@ async function uploadImageFile(projectDir: string) {
...
+      Body: fileStream,
+      ContentType: 'image/jpeg',
+      ACL: 'public-read',
+    };
+    await s3Client.upload(uploadParams).promise();
+  } catch (err) {
+    if (err.code !== 'AccessControlListNotSupported') {
+      throw err;
+    }
+  }
```
Comment (sobolk) [inline packages/amplify-e2e-tests/src/schema-api-directives/tests/predictions-usage.ts:72]: how is this helping?
if test is not sensitive to this upload then why don't we just remove this step altogether?

Comment (sundersc) [reply packages/amplify-e2e-tests/src/schema-api-directives/tests/predictions-usage.ts:72]: Currently this test is not validating anything after this file upload step. There is a backlog item to fix this.
This test is still useful to make sure a GraphQL schema with predictions directive can be compiled and pushed to cloud.

Comment (danielleadams) [reply packages/amplify-e2e-tests/src/schema-api-directives/tests/predictions-usage.ts:72]: I agree here - this is a duplicate test. Can we just delete this test? I think this adds more confusion if this doesn't end up getting "fixed" in the future than it does anything else

**Review by sobolk — APPROVED** [review]
Please merge if you have evidence that test passes after this change.

**Review by danielleadams — APPROVED** [review]
Approved as long as all refs are deleted and tests pass

**Pull request merged**